src/config/sidebar_config.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"menuLinks": [{"type": "item","link": "/jupyter/","text": "Notebooks","icon": "book"},{"type": "item","link": "/katib/","text": "Experiments (AutoML)","icon": "kubeflow:katib"},{"type": "item","text": "Experiments (KFP)","link": "/pipeline/#/experiments","icon": "done-all"},{"type": "item","link": "/pipeline/#/pipelines","text": "Pipelines","icon": "kubeflow:pipeline-centered"},{"type": "item","link": "/pipeline/#/runs","text": "Runs","icon": "maps:directions-run"},{"type": "item","link": "/pipeline/#/recurringruns","text": "Recurring Runs","icon": "device:access-alarm"},{"type": "item","link": "/volumes/","text": "Volumes","icon": "device:storage"},{"type": "item","link": "/tensorboards/","text": "Tensorboards","icon": "assessment"}],"externalLinks": [],"quickLinks": [{"text": "Upload a pipeline","desc": "Pipelines","link": "/pipeline/"},{"text": "View all pipeline runs","desc": "Pipelines","link": "/pipeline/#/runs"},{"text": "Create a new Notebook server","desc": "Notebook Servers","link": "/jupyter/new?namespace=kubeflow"},{"text": "View Katib Experiments","desc": "Katib","link": "/katib/"}],"documentationItems": [{"text": "Kubeflow Basics","desc": "Learn how to get started with Kubeflow","link": "https://charmed-kubeflow.io/docs/kubeflow-basics"},{"text": "Microk8s for Kubeflow","desc": "Quickly get Kubeflow running locally on native hypervisors","link": "https://microk8s.io/docs/addon-kubeflow"},{"text": "Requirements for Kubeflow","desc": "Get more detailed information about using Kubeflow and its components","link": "https://www.kubeflow.org/docs/started/requirements/"}]}
```

Comment:
```suggestion
{
    "menuLinks": [
        {
            "type": "item",
            "link": "/jupyter/",
            "text": "Notebooks",
            "icon": "book"
        },
        {
            "type": "item",
            "link": "/katib/",
            "text": "Experiments (AutoML)",
            "icon": "kubeflow:katib"
        },
        {
            "type": "item",
            "text": "Experiments (KFP)",
            "link": "/pipeline/#/experiments",
            "icon": "done-all"
        },
        {
            "type": "item",
            "link": "/pipeline/#/pipelines",
            "text": "Pipelines",
            "icon": "kubeflow:pipeline-centered"
        },
        {
            "type": "item",
            "link": "/pipeline/#/runs",
            "text": "Runs",
            "icon": "maps:directions-run"
        },
        {
            "type": "item",
            "link": "/pipeline/#/recurringruns",
            "text": "Recurring Runs",
            "icon": "device:access-alarm"
        },
        {
            "type": "item",
            "link": "/volumes/",
            "text": "Volumes",
            "icon": "device:storage"
        },
        {
            "type": "item",
            "link": "/tensorboards/",
            "text": "Tensorboards",
            "icon": "assessment"
        }
    ],
    "externalLinks": [
        
    ],
    "quickLinks": [
        {
            "text": "Upload a pipeline",
            "desc": "Pipelines",
            "link": "/pipeline/"
        },
        {
            "text": "View all pipeline runs",
            "desc": "Pipelines",
            "link": "/pipeline/#/runs"
        },
        {
            "text": "Create a new Notebook server",
            "desc": "Notebook Servers",
            "link": "/jupyter/new?namespace=kubeflow"
        },
        {
            "text": "View Katib Experiments",
            "desc": "Katib",
            "link": "/katib/"
        }
    ],
    "documentationItems": [
        {
            "text": "Kubeflow Basics",
            "desc": "Learn how to get started with Kubeflow",
            "link": "https://charmed-kubeflow.io/docs/kubeflow-basics"
        },
        {
            "text": "Microk8s for Kubeflow",
            "desc": "Quickly get Kubeflow running locally on native hypervisors",
            "link": "https://microk8s.io/docs/addon-kubeflow"
        },
        {
            "text": "Requirements for Kubeflow",
            "desc": "Get more detailed information about using Kubeflow and its components",
            "link": "https://www.kubeflow.org/docs/started/requirements/"
        }
    ]
}
```

file size diff is 2.64KB vs 1.57KB

but a way more readable

Reply:
This will not work in the configmap if I recall correctly

Reply:
It has problem with whitespaces

Reply:
but how do you read the string? just file readlines ?

Reply:
if you `json.load` from file into dict and then `json.dumps` , then it should work fine

Reply:
well if I recall correctly this will cause some error ... we can run it to get the error but I know I had to make it one line

Reply:
https://stackoverflow.com/questions/48632585/how-to-generate-json-using-jinja2-template-engine

from here looks to work

Reply:
will CI catch the error if that does not work ?

Reply:
@beliaev-maksim I think it is fair to have this conversation in a follow up PR, given this one is just for rearranging the components in the side panel.

Reply:
Let's track this in #101